src/types/person.rs

```diff
@@ -588,4 +614,90 @@ Claude Garamond",
         assert_eq!(p.suffix, "Sr.");
         assert_eq!(p.given_name, "Harcourt Fenton");
     }
+
+    #[test]
+    fn test_person_extended_name_format() {
+        // AUTHOR = {given=Hans, family=Harman}
+        let p = Person::parse(&[Spanned::zero(N("given=Hans, family=Harman"))]);
+        assert_eq!(p.name, "Harman");
+        assert_eq!(p.prefix, "");
+        assert_eq!(p.suffix, "");
+        assert_eq!(p.given_name, "Hans");
+
+        // AUTHOR = {given={Jean Pierre}}
+        let p =
+            Person::parse(&[Spanned::zero(N("given=")), Spanned::zero(V("Jean Pierre"))]);
+        assert_eq!(p.name, "");
+        assert_eq!(p.prefix, "");
+        assert_eq!(p.suffix, "");
+        assert_eq!(p.given_name, "Jean Pierre");
+
+        // AUTHOR = {given={Jean Pierre Simon}, given-i=JPS}
+        let p = Person::parse(&[
+            Spanned::zero(N("given=")),
+            Spanned::zero(V("Jean Pierre Simon")),
+            Spanned::zero(N(", given-i=JPS")),
+        ]);
+        assert_eq!(p.name, "");
+        assert_eq!(p.prefix, "");
+        assert_eq!(p.suffix, "");
+        assert_eq!(p.given_name, "Jean Pierre Simon");
+
+        // AUTHOR = {given=Jean, prefix=de la, prefix-i=d, family=Rousse}
+        let p = Person::parse(&[Spanned::zero(N(
+            "given=Jean, prefix=de la, prefix-i=d, family=Rousse",
+        ))]);
+        assert_eq!(p.name, "Rousse");
+        assert_eq!(p.prefix, "de la");
+        assert_eq!(p.suffix, "");
+        assert_eq!(p.given_name, "Jean");
+
+        // AUTHOR = {"family={Robert and Sons, Inc.}"}
+        let p = Person::parse(&[
+            Spanned::zero(N("family=")),
+            Spanned::zero(V("Robert and Sons, Inc.")),
+        ]);
+        assert_eq!(p.name, "Robert and Sons, Inc.");
+        assert_eq!(p.prefix, "");
+        assert_eq!(p.suffix, "");
+        assert_eq!(p.given_name, "");
+
+        // AUTHOR = {given=Hans, family=Harman and given=Simon, prefix=de, family=Beumont}
+        let people = &[Spanned::zero(N(
+            "given=Hans, family=Harman and given=Simon, prefix=de, family=Beumont",
+        ))];
+        let people: Vec<Person> = Type::from_chunks(people).unwrap();
+        assert_eq!(people.len(), 2);
+        assert_eq!(people[0].name, "Harman");
+        assert_eq!(people[0].prefix, "");
+        assert_eq!(people[0].suffix, "");
+        assert_eq!(people[0].given_name, "Hans");
+        assert_eq!(people[1].name, "Beumont");
+        assert_eq!(people[1].prefix, "de");
+        assert_eq!(people[1].suffix, "");
+        assert_eq!(people[1].given_name, "Simon");
+
+        // AUTHOR = {Hans Harman and given=Simon, prefix=de, family=Beumont}
+        let people =
+            &[Spanned::zero(N("Hans Harman and given=Simon, prefix=de, family=Beumont"))];
```

Comment:
Should `given = Simon, prefix = de, family = Beumont` also work? If so, please add a test!